src/themes/default/components/core/ProductBundleOptions.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <form class="custom-options">
     <div v-for="option in product.bundle_options" :key="('bundleOption_' + option.option_id)">
-      <product-bundle-option :option="option" @optionChanged="optionChanged" :error-messages="errorMessages" />
+      <product-bundle-option :option="option" @option-changed="optionChanged" :error-messages="errorMessages" />
     </div>
```

Comment:
Shouldn’t it be “bundleOptionChanged” as for the function name?

Reply:
I think that this is fine as it is. As it only Happens in files that have bundle in the name so it should be clear what is happening here.

When you think it is not so clear we can change it.